pkg/controlplane/apiserver/options/options.go

```diff
@@ -233,52 +241,91 @@ func (o *Options) Complete(alternateDNS []string, alternateIPs []net.IP) (Comple
 	// adjust authentication for completed authorization
 	completed.Authentication.ApplyAuthorization(completed.Authorization)
 
-	// verify and adjust ServiceAccountTokenMaxExpiration
+	err := o.completeServiceAccountOptions(ctx, &completed)
+	if err != nil {
+		return CompletedOptions{}, err
+	}
+
+	for key, value := range completed.APIEnablement.RuntimeConfig {
+		if key == "v1" || strings.HasPrefix(key, "v1/") ||
+			key == "api/v1" || strings.HasPrefix(key, "api/v1/") {
+			delete(completed.APIEnablement.RuntimeConfig, key)
+			completed.APIEnablement.RuntimeConfig["/v1"] = value
+		}
+		if key == "api/legacy" {
+			delete(completed.APIEnablement.RuntimeConfig, key)
+		}
+	}
```

Comment:
I think this bit is a bad rebase and should be dropped

Reply:
oops, nevermind, git made it show up as an addition in a confusing way when it wasn't